internal/migration/new/new.go

```diff
@@ -11,8 +11,16 @@ import (
 	"github.com/supabase/cli/internal/utils"
 )
 
-func Run(migrationName string, stdin afero.File, fsys afero.Fs) error {
-	path := GetMigrationPath(utils.GetCurrentTimestamp(), migrationName)
+func Run(repeatable bool, migrationName string, stdin afero.File, fsys afero.Fs) error {
+	path := ""
+
```

Comment:
```suggestion
	var path string
```

Nitpick: since the empty string is not used, it's more idiomatic to declare the variable for assignment later.